File: csharp/kitchen_manager/KitchenManager.cs
using System;

namespace KitchenManagerApp
{
    public class KitchenManager
    {
        private Chef chef = new Chef();
        private Dishwasher dishwasher = new Dishwasher();
        private Waiter waiter = new Waiter();

        public void Run()
        {
            chef.PrepareFood();
            waiter.ServeCustomers();
            dishwasher.WashDishes();
        }
    }
}

File: kotlin/src/kitchen_manager/KitchenManager.kt
package kitchen_manager

class KitchenManager {
    private val chef = Chef()
    private val dishwasher = Dishwasher()
    private val waiter = Waiter()

    fun run() {
        chef.prepareFood()
        waiter.serveCustomers()
        dishwasher.washDishes()
    }
}